crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -88,7 +88,43 @@ pub enum Error {
 /// }
 /// ```
 ///
-/// ## Basic configuration
+/// ## Pre-configured Tracing Instance
+///
+/// There are two different styles to configure a [`Tracing`] instance: Using an opinionated pre-
+/// configured instance or a fully customizable builder. The first option should be suited for
+/// pretty much all operators by using sane defaults and applying best practices out-of-the-box.
+/// [`Tracing::pre_configured`] lists details about environment variables, filter levels and
+/// defaults used.
+///
+/// ```
+/// use stackable_telemetry::tracing::{Tracing, TelemetryOptions, Error};
+///
+/// #[tokio::main]
+/// async fn main() -> Result<(), Error> {
+///     let options = TelemetryOptions {
```

Comment:
Maybe we want docs for `clap` usage too? Not sure what that would look like, and I wondered whether the docs should also be gated behind the `clap` feature.

Reply:
Done in 5fe969d0c05b29e6f40b4d14492d09240fd4fe1f.